extensions/mssql/src/contracts.ts

```diff
@@ -1108,7 +1108,7 @@ export namespace TableDesignerGenerateScriptRequest {
 }
 
 export namespace TableDesignerGenerateChangePreviewReportRequest {
-	export const type = new RequestType<azdata.designers.TableInfo, string, void, void>('tabledesigner/generatepreviewreport');
+	export const type = new RequestType<azdata.designers.TableInfo, string | { report: string, format: string }, void, void>('tabledesigner/generatepreviewreport');
```

Comment:
Is there any specific reason you want to support just the report as a single value? Seems like you might as well simplify it and have it always take an object and just have the format be an optional value.

Reply:
i was mainly think about making it easy if the provider only have plain text report. also if we make it a union type the current STS will work fine (not a compelling reason though). I am fine removing the string type from it and update STS first to return the new expected data type.